x-pack/plugins/enterprise_search/server/routes/app_search/credentials.ts

```diff
@@ -42,9 +23,30 @@ export function registerCredentialsRoutes({
     },
     enterpriseSearchRequestHandler.createRequest({
       path: '/as/credentials/collection',
-      hasValidData: (body?: ICredentialsResponse) => {
```

Comment:
I removed this from the routes because it simplifies this file and I'm not sure it was necessary.